googlesearch/__init__.py

```diff
@@ -165,40 +166,33 @@ def get_page(url, user_agent=None, verify_ssl=True):
     :raises urllib2.URLError: An exception is raised on error.
     :raises urllib2.HTTPError: An exception is raised on error.
     """
-    if user_agent is None:
-        user_agent = USER_AGENT
+    user_agent = USER_AGENT if user_agent is None else user_agent
     request = Request(url)
     request.add_header('User-Agent', user_agent)
     cookie_jar.add_cookie_header(request)
-    if verify_ssl:
-        response = urlopen(request)
-    else:
-        context = ssl._create_unverified_context()
-        response = urlopen(request, context=context)
+    :
+    response = urlopen(request) if verify_ssl urlopen(request, context=ssl._create_unverified_context())
     cookie_jar.extract_cookies(response, request)
     html = response.read()
     response.close()
     try:
         cookie_jar.save()
-    except Exception:
+    except Exception: <== # What exception gets raised here?
```

Comment:
Ideally, there'd be a logger to log these exceptions, and then after some testing those would be imported and specified here. Or a special exception class would be created for this module.

Reply:
It really doesn't matter since that exception only happens on some weird environments where the cookie jar can't be stored to disk.